plugins/woocommerce/src/Internal/Admin/Orders/Edit.php

```diff
@@ -104,6 +111,18 @@ private function enqueue_scripts() {
 		wp_enqueue_script( 'post' ); // Ensure existing JS libraries are still available for backward compat.
 	}
 
+	/**
+	 * Returns the PageController for this edit form. This method is protected to allow child classes to overwrite the PageController object and return custom links.
+	 *
```

Comment:
Needs `@since`

Reply:
Addressed in https://github.com/woocommerce/woocommerce/pull/39193/commits/0629e2fb6389f8958f70c61cbb25901dd496179d